classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -146,22 +183,31 @@ protected function process_bulk_action() {
 
 		check_admin_referer( 'bulk-' . $this->_args['plural'] );
 
-		$method   = 'bulk_' . $action;
+		$method = 'bulk_' . $action;
 		if ( array_key_exists( $action, $this->bulk_actions ) && is_callable( array( $this, $method ) ) && ! empty( $_GET['ID'] ) && is_array( $_GET['ID'] ) ) {
 			$ids_sql = '(' . implode( ',', array_fill( 0, count( $_GET['ID'] ), '%s' ) ) . ')';
-			$this->$method( $_GET['ID'], $wpdb->prepare( $ids_sql, $_GET['ID'] ) );
+			$id      = sanitize_text_field( wp_unslash( $_GET['ID'] ) );
```

Comment:
`$_GET['ID']` will be an array (we're dealing with bulk actions, so more than one can be passed), so we need to take a different approach as passing an array through `sanitize_text_field()` gives us an empty string and that means the bulk action will not be processed successfully.

Recommend:

```suggestion
			$id      = array_map( 'absint', $_GET['ID'] );
```